app/server.go

```diff
@@ -1046,7 +1042,9 @@ func (s *Server) Shutdown() {
 		s.Cluster.StopInterNodeCommunication()
 	}
 
-	s.StopMetricsServer()
+	if err = s.platform.ShutdownMetrics(); err != nil {
+		mlog.Warn("Failed to stop metrics server", mlog.Err(err))
```

Comment:
s.Log.Warn

Reply:
Yes, let's do that. I was thinking of keeping it to the scope of this PR, but we can change all. 0/5.